<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraView</class>
 <widget class="QWidget" name="CameraView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1,1">
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Light</family>
       <pointsize>11</pointsize>
       <weight>3</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 11pt &quot;Segoe UI Light&quot;;</string>
     </property>
     <property name="title">
      <string>Front Camera</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="20,1">
      <item row="0" column="0">
       <widget class="QLabel" name="CamPreview1">
        <property name="font">
         <font>
          <family>Segoe UI Light</family>
          <pointsize>11</pointsize>
          <weight>3</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="CamButton1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Click to Preview</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 25 11pt &quot;Segoe UI Light&quot;;</string>
     </property>
     <property name="title">
      <string>Welding Camera</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" rowstretch="20,1">
      <item row="0" column="0">
       <widget class="QLabel" name="CamPreview2">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="CamButton2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Click to Preview</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
